CHANGELOG.md

```diff
@@ -31,8 +33,7 @@
 
 **Other changes**
 
-- Provider root paths are evaluated relative to the location of the configuration 
-  file 
+- Provider root paths are evaluated relative to the location of the configuration file
```

Comment:
I did not see a typo here, do I miss something?
Generally text files should have a break at ~ 80 columns, although I admit that this is not always the case yet ;-)

Reply:
Sorry I'm a fan of wide screens :p It's a long debate but my editor is configured on columns of 120 (PEP8 is a bit too restrictive for me in 2022 :p) I'll fix this line in next commit :)